src/transferslot.cpp

```diff
@@ -1115,6 +1138,33 @@ void TransferSlot::doio(MegaClient* client)
     }
 }
 
+
+bool TransferSlot::tryRaidRecoveryFromHttpGetError(unsigned connectionNum)
+{
+    // If we are downloding a cloudraid file then we may be able to ignore one connection and download from the other 5.
+    if (transferbuf.isRaid())
```

Comment:
Since `RadiBufferManager::tryRaidHttpGetErrorRecovery()` is only used here and it already checks the `isRaid()`, I'd suggest to convert the checkup in the `RadiBufferManager::tryRaidHttpGetErrorRecovery()` to an assert.

Reply:
Yes, good point, and done.  cheers